setup.cfg

```diff
@@ -1,5 +1,39 @@
+[metadata]
+name = kaitaistruct
+version = 0.1.0
```

Comment:
Shall we have it here in the first place, given that we override it in the code? If it's mandatory, could we put something clearly invalid (like `???` or `xxx` or `0.0.0`) here?

Reply:
Oops, I have forgotten to remove it after I have copied my boilerplate.